grinder/defaultvalues.py

```diff
@@ -106,10 +111,23 @@ class DefaultVulnersScanValues:
     VULNERS_SCRIPT_PATH = "/plugins/vulners.nse"
 
 
+class DefaultMasscanScanValues:
+    """
+    Default values for Masscan scan itself
+    """
+
+    PORTS = None
+    TOP_PORTS = None
+    RATE = 5000
```

Comment:
Are you sure about it?

Rate 5k, network 87.250.250.96/24, 54 results:
```bash
⭕ Current scan host (0/54): 87.250.250.1:53 (started at: 19:47:26)
⭕ Current scan host (1/54): 87.250.250.107:443 (started at: 19:47:26)
⭕ Current scan host (3/54): 87.250.250.111:443,80 (started at: 19:47:26)
⭕ Current scan host (2/54): 87.250.250.110:443 (started at: 19:47:26)
⭕ Current scan host (4/54): 87.250.250.120:443 (started at: 19:47:26)
⭕ Current scan host (5/54): 87.250.250.126:80 (started at: 19:47:26)
⭕ Current scan host (6/54): 87.250.250.127:80 (started at: 19:47:26)
⭕ Current scan host (7/54): 87.250.250.131:443 (started at: 19:47:26)
⭕ Current scan host (9/54): 87.250.250.138:80 (started at: 19:47:26)
⭕ Current scan host (8/54): 87.250.250.132:443 (started at: 19:47:26)
⭕ Current scan host (10/54): 87.250.250.139:80 (started at: 19:47:26)
```
Rate 500, network 87.250.250.96/24, 80 results:
```bash
⭕ Current scan host (0/80): 87.250.250.1:53 (started at: 20:10:00)
⭕ Current scan host (1/80): 87.250.250.107:443,80 (started at: 20:10:00)
⭕ Current scan host (2/80): 87.250.250.110:443,80 (started at: 20:10:00)
⭕ Current scan host (3/80): 87.250.250.111:80,443 (started at: 20:10:00)
⭕ Current scan host (4/80): 87.250.250.113:53 (started at: 20:10:00)
⭕ Current scan host (5/80): 87.250.250.114:443 (started at: 20:10:00)
⭕ Current scan host (6/80): 87.250.250.115:443 (started at: 20:10:00)
⭕ Current scan host (7/80): 87.250.250.119:443,80 (started at: 20:10:00)
⭕ Current scan host (8/80): 87.250.250.12:80,443 (started at: 20:10:00)
⭕ Current scan host (9/80): 87.250.250.120:443,80 (started at: 20:10:00)
⭕ Current scan host (10/80): 87.250.250.122:443,80 (started at: 20:10:00)
```
Maybe it's better to reduce default to 1.000 packets?

Reply:
You are right. In any case, user can change the default rate value in the query file if he wants. I will fix it.